Microsoft.Toolkit.Uwp.UI.Controls/ControlHelpers.XamlHost.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Internal class used to provide helpers for controls
+    /// </summary>
+    internal static partial class ControlHelpers
+    {
+        private static bool isXamlRootAvailable = Windows.Foundation.Metadata.ApiInformation.IsPropertyPresent("Windows.UI.Xaml.UIElement", "XamlRoot");
+        private static bool isXamlRootAvailableInitialized = false;
+
+        internal static bool IsXamlRootAvailable
+        {
+            get
+            {
+                if (!isXamlRootAvailableInitialized)
+                {
+                    InitializeXamlRootAvailable();
+                }
+
+                return isXamlRootAvailable;
+            }
+        }
+
+        internal static void InitializeXamlRootAvailable()
+        {
+            isXamlRootAvailable = Windows.Foundation.Metadata.ApiInformation.IsPropertyPresent("Windows.UI.Xaml.UIElement", "XamlRoot");
```

Comment:
>isXamlRootAvailable [](start = 12, length = 19)

This seems odd - isXamlRootAvailable is initialized here and above where it's declared, doesn't seem like it should be necessary twice.
Why not just use a static ctor?  That should be guaranteed to run before the property getter can be used by anyone - then you wouldn't need the "initialized" variable as well.

Reply:
I wanted to lazy load this, like we have at other placing using this patter with ApiInformation, but this code path is so common that I optimized it there and forgot to remove it here. I'll push a fix!

Reply:
Addressed by https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3206/commits/d2ea666ddce8a285e3abd80bbc8f85cc794a2e1b